### PR TITLE
Support `password_override` option

### PR DESCRIPTION
**Pull request by yuhiwa**
This pull request introduces a new option password_override to ssh operator like pg operator.
treasure-data/digdag#839

Thank you for this advice.
https://github.com/hiroyuki-sato/digdag-plugin-ssh/pull/3#issuecomment-416168633

### CODE REVIEW

File: src/main/java/com/github/hiroyuki_sato/digdag/plugin/SshOperatorFactory.java [hunk 1]
```diff
@@ -133,21 +133,21 @@ private void authorize(SSHClient ssh)
                     request.getConfig().getNestedOrGetEmpty("ssh"));
 
             String user = params.get("user", String.class);
-            SecretProvider secret = context.getSecrets().getSecrets("ssh");
+            SecretProvider secrets = context.getSecrets().getSecrets("ssh");;
```
Comment (hiroyuki-sato) [inline src/main/java/com/github/hiroyuki_sato/digdag/plugin/SshOperatorFactory.java]: `;;` -> `;`

**Comment by hiroyuki-sato** [comment]
@yuhiwa 

Thank you for the great work!.
I'll take a look later. Please wait. 

If you are interested in maintaining the project, I grant you the permissions(committer).
What do you think?

このプラグイン今後も利用する予定があって興味があればメンテナになりますか？

**Comment by yuhiwa** [comment]
I'm very interested in it and I'm going to be using this plugin. I'd like to get the permissions.

はい、このプラグインを今後も利用する予定で興味があるのでよろしくおねがいします。

**Comment by hiroyuki-sato** [comment]
@yuhiwa 

Sorry late reply, LGTM 👍

**Pull request merged**